crates/nu-protocol/src/ast/expr.rs

```diff
@@ -30,7 +30,7 @@ pub enum Expr {
     MatchBlock(Vec<(MatchPattern, Expression)>),
     List(Vec<Expression>),
     Table(Vec<Expression>, Vec<Vec<Expression>>),
-    Record(Vec<(Expression, Expression)>),
+    Record(Vec<RecordItem>),
```

Comment:
It bugs me a little that there's an `Expr::Spread` variant, but it's not used for records, just lists. Would it be a good idea to make `Expr::List` also accept instances of some `ListItem` enum for consistency? It would mean refactoring a lot of code, though.